static/app/components/group/releaseStats.tsx

```diff
@@ -30,17 +30,14 @@ function GroupReleaseStats({
   group,
   currentRelease,
 }: Props) {
-  const environment =
-    environments.length > 0
-      ? environments.map(env => env.displayName).join(', ')
```

Comment:
AFAICT, `displayName` is not a real property and environments only have a `name`, so just using an array of env names should be the same as doing this. Please correct me if there is something to this whole displayName thing.